package.json

```diff
@@ -106,6 +106,7 @@
       "express": "^4.19.2",
       "follow-redirects": "^1.15.6",
       "glob-parent": "^5.1.2",
+      "jsdom": "^24.0.0",
```

Comment:
`jsdom` had two dependencies which were deprecated so need to override it to latest version. Also, this won't affect the package as `jsdom` is only used by `jest-environment-jsdom`.